var/spack/repos/builtin/packages/gearshifft/package.py

```diff
@@ -54,4 +54,5 @@ def cmake_args(self):
             '-DGEARSHIFFT_CLFFT:BOOL={0}'.format(
                 'ON' if '+clfft' in spec else 'OFF')
         ])
+        args.append('-DCMAKE_CXX_FLAGS=-pthread')
```

Comment:
Is this true in general or tied to a specific version of CMake?

Reply:
I think it's a general version of CMake.

Reply:
I'll change this to only work with %gcc.

Reply:
I changed this only work with %gcc.

Reply:
This change is for the general version.